erpnext/regional/doctype/gst_hsn_code/gst_hsn_code.js

```diff
@@ -3,6 +3,26 @@
 
 frappe.ui.form.on('GST HSN Code', {
 	refresh: function(frm) {
-
+		if(! frm.doc.__islocal && frm.doc.taxes.length){
+			frm.add_custom_button(__('Update Taxes for Items'), function(){
+				frappe.confirm(
+					'Are you sure? It will overwrite taxes for all items.',
```

Comment:
It will overwrite taxes for all items with HSN Code {0}

Reply:
Done